ui/lib/core/addon/utils/sanitize-path.js

```diff
@@ -4,10 +4,40 @@
  */
 
 export function sanitizePath(path) {
+  if (!path) return '';
   //remove whitespace + remove trailing and leading slashes
   return path.trim().replace(/^\/+|\/+$/g, '');
 }
 
 export function ensureTrailingSlash(path) {
   return path.replace(/(\w+[^/]$)/g, '$1/');
 }
+
+/**
+ * getRelativePath is for removing matching segments of a subpath from the front of a full path
+ * @param {string} fullPath eg apps/prod/app_1/test
+ * @param {string} rootPath eg apps/prod
+ * @returns the leftover segment, eg app_1/test
+ */
+export function getRelativePath(fullPath = '', rootPath = '') {
```

Comment:
I initially had this logic within the namespace computed property, but it was much easier to test as a util